README.md

```diff
@@ -1,48 +1,208 @@
 [![Maven Package upon a push](https://github.com/mosip/pre-registration/actions/workflows/push-trigger.yml/badge.svg?branch=release-1.3.x)](https://github.com/mosip/pre-registration/actions/workflows/push-trigger.yml)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?branch=release-1.3.x&project=mosip_pre-registration&metric=alert_status)](https://sonarcloud.io/dashboard?branch=release-1.3.x&id=mosip_pre-registration)
 
-# Pre-registration 
-This repository contains the source code and design documents for MOSIP Pre-registration server.  For an overview refer [here](https://docs.mosip.io/1.2.0/modules/pre-registration).  The modules exposes API endpoints. For a reference front-end UI implementation refer to [Pre-registration UI github repo](https://github.com/mosip/pre-registration-ui/). Pre-registration Developers Guide [here](https://docs.mosip.io/1.2.0/modules/pre-registration/pre-registration-developer-setup)
+## Overview
+
+The **Pre-Registration** module enables a resident to:
+
+- Enter demographic data and upload supporting documents.
+- Book an appointment for one or many users by choosing a suitable registration center and a convenient time slot
+- Receive appointment notifications.
+- Reschedule and cancel appointments.
+
+Once the resident completes the above process, their data is downloaded at the respective registration centers before their appointment, thus, saving enrollment time. The module supports multiple languages.
+
+It exposes a set of APIs that enables resident to do Pre-Registration activities efficiently. A reference front-end implementation is available in the [Pre-registration UI](https://github.com/mosip/pre-registration-ui/). 
+
+For a complete functional overview and capabilities, refer to the [official documentation](https://docs.mosip.io/1.2.0/modules/pre-registration).
+
+Pre-registration Developers Guide [here](https://docs.mosip.io/1.2.0/modules/pre-registration/pre-registration-developer-setup)
+
+## Services
 
 Pre-registration module consists of the following services:
```

Comment:
Application - This service helps to create a Pre Registration application for a resident and it also uses Pre Registration Booking service (give repo link) to book appointment.
Batchjob - This cron job runs as per the schedule and helps is many background tasks like creating appointment slots for Registration Centres, archiving  Pre Registration applications once workflow is over etc
Core - This service contains many reusable components across all Pre Registration services
Datasync - This service syncs Pre Registration application data with Registration Centre. This service also marks Pre Registration application as consumed (reverse sync) from Registration Processor once the workflow is complete.